app/views/components/_expandable_content.html.erb

```diff
@@ -2,22 +2,33 @@
   start_expanded ||= false
   open_label ||= "More details"
   close_label ||= "Fewer details"
+  button_at_top ||= false
+
+  toggle_button = content_tag(:div, class: "form-group js-expand-button-wrapper hidden") do
```

Comment:
This might fit better in a helper?

Reply:
@pmanrubia I'd actually thought about just moving it into a partial. The reason I didn't was that having a partial (or even a helper) would make this feel much less like a standalone component (which is what it's trying to be).

I have no strong feelings, though, so if you'd prefer, I can move this code into a partial?